src/DynamoCore/DSEngine/EngineController.cs

```diff
@@ -43,7 +40,7 @@ private void OnTraceReconciliationComplete(TraceReconciliationEventArgs e)
             }
         }
 
-        private readonly LiveRunnerServices liveRunnerServices;
+        public readonly LiveRunnerServices liveRunnerServices;
```

Comment:
Why do we need to make this public, is this for future work? I can't find references to it in this PR, and the naming doesn't follow the right scheme for a public API.